ingest-app/src/cmr/ingest/config.clj

```diff
@@ -5,6 +5,11 @@
             [cmr.oracle.connection :as conn]
             [cmr.message-queue.config :as rmq-conf]))
 
+(defconfig ingest-accept-umm-version
```

Comment:
I can not do that. Sorry for the misleading doc string. I was thinking of changing that but not sure exactly how to describe it.  Basically it's not really one version below, it's the highest official version, which could include the current-version.  
Also, we want to make this configurable because we want to allow the development/testers to be able to still ingest up to the current-version.